src/reason_pprint_ast.ml

```diff
@@ -630,9 +630,6 @@ let rules = [
     (TokenPrecedence, (fun s -> (Right, s = "asr")));
   ];
   [
-    (TokenPrecedence, (fun s -> (Left, s.[0] == '#')));
```

Comment:
We wanted `PREFIXOP` and `BANG` to have the exact same precedence as each other (but only in *some* use cases of `BANG` - not others). The only way to do that that I could see was to create a custom `%prec` and then make both of them use that custom precedence.